tests/dnfserver/container/generate-certs.sh

```diff
@@ -0,0 +1,28 @@
+#~/bin/bash
+mkdir -p certificates
+cd certificates
+rm *
+
+
+# todo - extend lifespan of all certs to 10 years
+
+# generate CA cert
+openssl genrsa   -out myCA.key 4096
+openssl req -new -x509  -key myCA.key -days 3650 -out myCA.crt \
```

Comment:
Can we use something like 36500 just to be safe we won't ever need to regenerate just because of validity :) ? Do we need to track this file though? You've clearly used it to generate certificates, but is it something we're going to be doing on a regular basis?

Reply:
no idea how often we would use it, IMO tracking files is free and if anyone ever needed to regenerate the certs this makes sure it is fast.

Reply:
changed to 36500 though